DEPLOYMENT.md

```diff
@@ -2,6 +2,32 @@
 
 Host on Heroku. [Integrated with GitHub](https://devcenter.heroku.com/articles/github-integration).
 
+## Prepare your environment
+
+```
+cd your_projects_folder
+git clone https://github.com/rubytaiwan/rubytw-reboot.git
+heroku login
+heroku git:remote -a rubytw-staging -r staging
+heroku git:remote -a rubytw-production -r production
+git remote -v
+# you will got staging/production remote name now.
+```
+
+## Local development environment
+
+Use `foreman` run as same Procfile environment.
+
+```
+$ gem install foreman
+# Do not install in Gemfile.
+```
+
+```
+$ foreman start
+# No more "rails s", do "foreman start" in your local development.
+```
```

Comment:
Could you also move "how to use foreman to run our application" into CONTRIBUTIND.md? Because DEPLOYMENT only documents things about deploy. :smiley: